i18n/en/docusaurus-plugin-content-docs/current/guides/tech/with-nuxtjs.mdx

```diff
@@ -0,0 +1,121 @@
+---
+sidebar_position: 10
+---
+# Use with NuxtJS
+
+It is possible to implement FSD in a NuxtJS project, but conflicts arise due to differences between the NuxtJS project structure requirements and FSD principles: 
+
+- Initially, NuxtJS offers a project file structure without a `src` folder, i.e. in the root of the project.
+- The file routing is in the `pages` folder, while in FSD this folder is reserved for the flat slice structure.
+
+
+## Adding an alias for the `rc` directory
```

Comment:
```suggestion
## Adding an alias for the `src` directory
```

Reply:
suggestion (non-blocking): Also, I personally think that aliases are not necessary for using Nuxt with FSD successfully, they are just convenient, with or without FSD. In this way, maybe we shouldn't bother mentioning them in this guide